### DISCUSSION TITLE
Getting images from computer to show on site published on github.

### DISCUSSION THREAD

**Discussion by Titan3T** [community/community#21629] [New to GitHub]
<p>I am having trouble getting an image to show up once I publish my site on github. The image is located on my computer. </p>
<table><tbody>
<tr><td>
<span>&lt;td&gt;</span><span>&lt;img <span>src</span>="<a target="_blank">C:\Users\ETam_\OneDrive\Desktop\web development\HTML - personal site\images\selfie.JPG</a>" <span>alt</span>="<span>selfie</span>"&gt;&lt;/td&gt;</span>
</td></tr>
<tr>
<td> </td>
<td> </td>
</tr>
</tbody></table>
<p>how would I go about fixing my broken image?  Thanks.</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/titan3t">@titan3t</a>,</p>
<p>Welcome to the GitHub Community forum!</p>
<p>Did you try to upload this image in a github repository? If you this you can reffer to it by something like this:</p>
<pre><code>&lt;img src="https://github.com/Titan3T/&lt;name_of_the_repository&gt;/blob/master/&lt;name_of_the_image&gt;" alt="selfie"&gt;
</code></pre>
<p>Hope this helps :slightly_smiling_face:</p>
<p>-Gabriele-</p>
<blockquote>
<p><strong>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!</strong></p>
</blockquote>

**Comment by Titan3T** [comment]
<p>Hello,</p>
<p>Thank you for your help. I was able to get my picture to show. Even something so simple took me a bit of time to figure out.</p>

**Comment by hassan647** [comment]
<p>uploading on github and using a local link for your image will not work you have to upload the link on repositories in order to make that link work</p>

**Comment by be-swansun** [comment]
<p>I’ve tried the above suggestion and it’s still not working!</p>

**Comment by missvickyy** [comment]
<p>Did it work for you? I can’t make it work. I tried a bunch of dif things.</p>

**Comment by missvickyy** [comment]
<p>How do I get that link Path? and does it need to be a new repository or uploading it to the same one that’s not displaying the image?</p>
